experiment/runner.py

```diff
@@ -65,9 +65,15 @@
 
 
 def _clean_seed_corpus(seed_corpus_dir):
-    """Moves seed corpus files from sub-directories into the corpus directory
-    root. Also, deletes any files that exceed the 1 MB limit."""
-    if not os.path.exists(seed_corpus_dir):
+    """Prepares |seed_corpus_dir| for the trial. This ensures that it can be
+    used by AFL which is picky about the seed corpus. Moves seed corpus files
+    from sub-directories into the corpus directory root. Also, deletes any files
+    that exceed the 1 MB limit. If the NO_SEEDS env var is specified than the
+    seed corpus files are deleted."""
+    if environment.get('NO_SEEDS'):
+        logs.info('NO_SEEDS specified, deleting seed corpus files.')
+        shutil.rmtree(seed_corpus_dir)
```

Comment:
Does seed_corpus_dir always exist ?

Reply:
It does, but I should probably added back this check right?

Reply:
Probably good to check or assert.